src/rich_click/cli.py

```diff
@@ -0,0 +1,51 @@
+"""The command line interface."""
+
+import sys
+from importlib import import_module
+
+try:
+    from importlib.metadata import entry_points
+except ImportError:
+    from importlib_metadata import entry_points
+
+import click
+from rich_click import group, command
+
+
+def main(args=None):
+    args = args or sys.argv[1:]
+    if not args:
+        # without args we assume we want to run rich-click on itself
+        # TODO: rewrite using click
+        script_name = "rich-click"
+    else:
+        script_name = args[0]
+    scripts = {script.name: script for script in entry_points().get("console_scripts")}
+    if script_name in scripts:
+        # a valid script was passed
+        script = scripts[script_name]
+        module_path = script.module
```

Comment:
Testing with [MultiQC](https://github.com/ewels/MultiQC) (before or after the point that I added rich-click support directly) I get this error:

```python
Traceback (most recent call last):
  File "/opt/miniconda3/envs/py3.8/bin/rich-click", line 33, in <module>
    sys.exit(load_entry_point('rich-click', 'console_scripts', 'rich-click')())
  File "/Users/phil/GitHub/ewels/rich-click/src/rich_click/cli.py", line 27, in main
    module_path = script.module
AttributeError: 'EntryPoint' object has no attribute 'module'
```

MultiQC has [a lot of entry points](https://github.com/ewels/MultiQC/blob/912f512d36459f147b1eac119c1bcea5f0594335/setup.py#L68-L210), which I guess is causing an issue..

Reply:
I get the same error with other packages too, and in fact with `rich-click` itself:

```console
$ rich-click --help
usage: rich-click [SCRIPT | MODULE:FUNCTION] [-- SCRIPT_ARGS...]

$ rich-click
Traceback (most recent call last):
  File "/opt/miniconda3/envs/py3.8/bin/rich-click", line 33, in <module>
    sys.exit(load_entry_point('rich-click', 'console_scripts', 'rich-click')())
  File "/Users/phil/GitHub/ewels/rich-click/src/rich_click/cli.py", line 27, in main
    module_path = script.module
AttributeError: 'EntryPoint' object has no attribute 'module'
```

Reply:
Weird, `module` should always be available: https://docs.python.org/3/library/importlib.metadata.html#entry-points
What Python version are you using?

Reply:
Ah, it's not in 3.8 indeed. I'll use `value` directly then.